core/network/besu/contracts/interop/.gitignore

```diff
@@ -0,0 +1 @@
+*.proto
```

Comment:
I realise that you won't be copying `.proto` files here but `.sol` files of compiled protos right? So may be you can copy the compiled protos into one directory say `protos-sol`, and add that directory to `.gitignore`?